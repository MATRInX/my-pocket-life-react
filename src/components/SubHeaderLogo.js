import React from 'react';
import { Link } from 'react-router-dom';
import LINKS from '../fixtures/links';

const SubHeaderLogo = () => (
  <div>
    <Link to={LINKS.HOME}>
      <img src="/images/mypocketlifev2.png" />
    </Link>
  </div>
);

export default SubHeaderLogo;