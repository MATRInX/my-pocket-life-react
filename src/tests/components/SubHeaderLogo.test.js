import React from 'react';
import { shallow } from 'enzyme';
import SubHeaderLogo from '../../components/SubHeaderLogo';

test('should correctly render SubHeaderLogo', () => {
  const wrapper = shallow(<SubHeaderLogo />);
  expect(wrapper).toMatchSnapshot();
});