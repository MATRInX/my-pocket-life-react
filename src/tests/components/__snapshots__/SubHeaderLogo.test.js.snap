// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render SubHeaderLogo 1`] = `
<div>
  <Link
    replace={false}
    to="/"
  >
    <img
      src="/images/mypocketlifev2.png"
    />
  </Link>
</div>
`;
